### ISSUE TITLE
B/L/XL configuration

### ISSUE THREAD

**Issue by stefenmax** [zhuyu-cs/MeanFlow#9]
Thanks for your excellent work! 
In Readme training configurations part. Seems you provide the B4 B2 L2, but without XL configuration.
And the pretrained weights link seems 404 error now.
Can you fix that? thanks

**Comment by stefenmax** [comment]
Thanks for your reply. I also have questions about the loss.In [train.py](https://github.com/zhuyu-cs/MeanFlow/blob/e4fd0e0948df7bad8b2e888f7e5ce040e5d65e4e/train.py#L225) what you get the loss is a tuple (loss, loss_mean_ref), not a single tensor. Calling .mean() on a tuple will raise an error because tuples don’t have a .mean() method.
And if I just extract the first value, the loss will remain1 and you mentioned it's normal when set p=1. The grad_norm value is around 5-20. Is it normal that this situation happens during the whole training? thanks!

**Comment by zhuyu-cs** [comment]
Yes, this phenomenon is completely normal. Although the loss value remains constant at 1, the grad_norm is not fixed.
